server/tables/tables.go

```diff
@@ -610,6 +610,18 @@ func PreAutoMigrate(db *gorm.DB) ([]PostAutoMigrateLogic, error) {
 			log.Warningf("Unsupported sql dialect: %q", db.Dialector.Name())
 		}
 	}
+	if m.HasTable("TargetStatuses") && m.HasIndex("TargetStatuses", "target_status_invocation_pk") {
+		// Drop the invocation_pk index; and set default to invocation_pk column.
+		postMigrate = append(postMigrate, func() error {
```

Comment:
do you want to condition the default column value on the index existing or treat them separately?

Reply:
I think it's fine. As long as on-prem users don't drop the index by hand.